### PR TITLE
Blog app - View

### PR DESCRIPTION
**Pull request by aliabbani**
## Blog app - Views

In this milestone I:

- Implemented the design from the sneak peek wireframes.
- Added styling to the app using bootstrap.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
-    <p><%= post.text %></p>
-    <div>
-      <span>Comments: <%= post.comments_counter%>,</span> 
-      <span>Likes: <%= post.likes_counter%></span>
-    </div>
-  <% end %>
+<div class="centro-user-show-button">
+  <a>
+    <%= link_to "See all posts", user_posts_path(@user), :class => "btn btn-primary" %>
+  </a>
```
Comment (DeliceLydia) [inline app/views/users/show.html.erb:28]: - [x] When I click on the `see all posts` button I should be able to see all user posts but currently I keep getting only 3recent posts please work on that

Comment (DeliceLydia) [inline app/views/users/show.html.erb]: - [ ] I suggest that you make this user page looks like the given wireframe
flame

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @aliabbani, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Major highlights for your project:
- Kudos to the good README
- Followed git workflow
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### Keep Rocking!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (aliabbani) [reply app/views/users/show.html.erb:28]: Done :white_check_mark:

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @aliabbani,

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**